### DISCUSSION TITLE
A less "manual" way of disabling kiosk mode?

### DISCUSSION THREAD

**Discussion by fcassata** [NemesisRE/kiosk-mode#250] [Q&A] [answered]
?disable_km works well of course, but when I'm developing a dashboard it becomes quite painful to have to constantly go back and forth that way. Especially when testing on the companion app on mobile, or on tablets, where I have to disable it in the yaml since there is no location bar.

Is there, or could there be a way to disable it by for example listening to an input_boolean? It's my understanding that I can't use conditional code in the top level file where the kiosk mode settings go.

**Comment by fcassata** [comment]
Maybe to put it more clearly, I'm looking for a way to disable kiosk mode from the actual lovelace UI.

**Reply by NemesisRE** [reply, marked as answer]
Just use [entity settings](https://github.com/NemesisRE/kiosk-mode?tab=readme-ov-file#entity_settings)

**Reply by fcassata** [reply]
I can't believe I missed that! Thank you so much for the reply, and the module!
